contracts/tgrade-valset/src/state.rs

```diff
@@ -139,3 +141,26 @@ impl<'a> IndexList<OperatorInfo> for OperatorIndexes<'a> {
         Box::new(v.into_iter())
     }
 }
+
+/// Export state
+pub fn export(deps: Deps<TgradeQuery>) -> Result<Response<TgradeMsg>, ContractError> {
+    // TODO
+    let export = ExportImport {
+        version: get_contract_version(deps.storage)?.version,
+        state: Empty {},
+    };
+
+    Ok(Response::new().set_data(to_binary(&export)?))
+}
+
+/// Import state
+pub fn import(
+    deps: DepsMut<TgradeQuery>,
+    import: ExportImport,
+) -> Result<Response<TgradeMsg>, ContractError> {
+    if import.version > get_contract_version(deps.storage)?.version {
```

Comment:
This fails for me with  `cw2::ContractVersion not found: execute wasm contract failed`

Reply:
This is because contract version is set during instantiation. Will remove the version check during import, but leave the version info field for info during export.